package.json

```diff
@@ -93,7 +97,7 @@
     "dev": "run-p dev:*",
     "dev:css": "yarn generate:css -- --watch",
     "dev:remix": "remix dev",
-    "dev:svg": "yarn svgr --silent --template svgr/template.js --filename-case kebab --out-dir ./app/ui/design-system/images ./app/ui/design-system/images",
+    "dev:svg": "chokidar --silent --initial \"./app/ui/design-system/images/**/*.svg\" -c \"yarn svgr --silent --template svgr/template.js --filename-case kebab --out-dir ./app/ui/design-system/images ./app/ui/design-system/images\"",
```

Comment:
this makes it so that the svgs are "watched" and also let me run this app through a Procfile